functional/lcow_test.go

```diff
@@ -126,17 +114,7 @@ func TestLCOWSimplePodScenario(t *testing.T) {
 	defer os.RemoveAll(c2ScratchDir)
 	c2ScratchFile := filepath.Join(c2ScratchDir, "sandbox.vhdx")
 
-	opts := &uvm.UVMOptions{
-		OperatingSystem: "linux",
-		ID:              "uvm",
-	}
-	lcowUVM, err := uvm.Create(opts)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if err := lcowUVM.Start(); err != nil {
-		t.Fatal(err)
-	}
+	lcowUVM := testutilities.CreateLCOWUVM(t, "uvm")
```

Comment:
I guess this could be `CreateLCOWUVMFromOpts(t, &uvm.UVMOptions{ID:"uvm"})` and have one less entry point? I'm not even sure we need a name anyway as I think it's defaulted unless that code has been removed.

Reply:
Actually I will do this as the next cleanup as we change the opts type and then it makes more sense to have this I think. Because it turns into:

`CreateLCOWUVMFromOpts(t, &uvm.OptionsLCOW{ Options: &uvm.Options{ ID: "uvm" } })`

Reply:
SGTM. Feel free to merge this, or add to as you want for the next cleanup.